cirq-core/cirq/protocols/channel.py

```diff
@@ -131,11 +131,11 @@ def channel(
         value is returned.
 
     Raises:
-        TypeError: `val` doesn't have a _channel_ or _unitary_ method (or that
+        TypeError: `val` doesn't have a _kraus_ or _unitary_ method (or that
             method returned NotImplemented) and also no default value was
             specified.
     """
-    channel_getter = getattr(val, '_channel_', None)
+    channel_getter = getattr(val, '_kraus_', None)
```

Comment:
`channel_getter` -> `kraus_getter`

Similarly for other local variables named `.*channel.*` in this file.

Reply:
Done.